examples/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ foreach(file-path ${test-sources})
     string( REPLACE ".cpp" "" file-path-without-ext ${file-path} )
     get_filename_component(file-name ${file-path-without-ext} NAME)
     add_executable( ${file-name} ${file-path})
-    target_link_libraries( ${file-name} PUBLIC unifex)
+    target_link_libraries(${file-name} PUBLIC unifex gtest_main)
```

Comment:
I assume linking in gtest_main treat the `main()` it provides as a weak symbol and still prefer the `main()` from the sources?

Reply:
I _think_ so.